designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/DeploymentManager.scala

```diff
@@ -27,8 +27,17 @@ trait DeploymentManager extends AutoCloseable {
 
   def test[T](name: ProcessName, canonicalProcess: CanonicalProcess, scenarioTestData: ScenarioTestData, variableEncoder: Any => T): Future[TestResults[T]]
 
+  /**
+    * Gets status from the engine
+    */
   def getProcessState(name: ProcessName)(implicit freshnessPolicy: DataFreshnessPolicy): Future[WithDataFreshnessStatus[Option[ProcessState]]]
 
+  /**
+    * Gets status from engine and resolves inconsistency with lastAction.
+    * ObsoleteStateDetector.handleObsoleteStatus
```

Comment:
1. We don't have `ObsoleteStateDetector` anymore. I would write:
```
Gets status from engine, resolves possible inconsistency with lastAction and formats status using `ProcessStateDefinitionManager`
```
2. We have to write in MigrationGuide that logic of state inconsistency detection was moved from designer to `DeploymentManager` - if you have this case, you should invoke it on your side

Reply:
Please remember about 2. - changes in MigrationGuide

Reply:
Fixed.